modules/juce_core/system/juce_TargetPlatform.h

```diff
@@ -63,7 +63,7 @@
 #elif defined (JUCE_ANDROID)
   #undef        JUCE_ANDROID
   #define       JUCE_ANDROID 1
-#elif defined (LINUX) || defined (__linux__)
+#elif defined (linux) || defined (__linux__)

```

Comment:
Are you sure about this? I've definitely seen the upper-case version being used!

Reply:
I looked for on  page https://sourceforge.net/p/predef/wiki/OperatingSystems/ .

Reply:
Where did you see upper-case version ?

Reply:
and here: http://www.faqs.org/docs/Linux-HOWTO/GCC-HOWTO.html

Reply:
OK, that sounds pretty convincing, but is there a reason to actually remove the upper-case one? I'd rather leave it in, just in case removing it would break people's builds.

Reply:
(Can't remember where I saw it used.. this was years ago..)

Reply:
Because this macro is badly.

Reply:
I'm confused - why would a BSD build have that macro set?

Reply:
I am sorry this is my mistake in my BSD build, but i think that macro does not exist

Reply:
OK. Thanks for the other suggestions though, I'll add those BSD includes for you!